Modelica/package.mo

```diff
@@ -1,5 +1,5 @@
 within ;
-package Modelica "Modelica Standard Library - Version 4.0.0"
+package Modelica "Modelica Standard Library - Version 4.1.0"
```

Comment:
I think we should just remove the " - Version 4.1.0". The information is readily available elsewhere, and it only adds to the maintenance burden.